libr/core/cmd_search.c

```diff
@@ -3343,6 +3341,9 @@ static int cmd_search(void *data, const char *input) {
 			{
 				RSearchKeyword *kw;
 				kw = r_search_keyword_new_hexmask ("00", NULL);
+				// AES search is at most 40 bytes
+				kw->keyword_length = 40;
```

Comment:
if there are not already, please define these constant values somewhere appropriate and use a meaningful name instead of hardcoding values like `40` or `14`.

Reply:
`AES_SEARCH_LENGTH`?